requirements.conda.yml

```diff
@@ -24,13 +24,15 @@ dependencies:
   - pandas
   - pillow
   - pip
+  - pydicom # Used by wsidicom
   - pixman>=0.39.0
```

Comment:
In requirements.dev.conda.yml it's pixman!=0.38.0

Reply:
Just checked develop and it is that same as here. Where are you seeing this difference?

Reply:
Hmm yes, I changed that because @shaneahmed (although I think that should really be a different PR as this one did not modify any constraints but only added the DICOM dependencies).

Reply:
I think >= 0.39 is a bit better than != 0.38, but either will work. Just need to be consistent.

Reply:
ok let's keep this consistent, I am ok with `>=0.39`